exercises/isbn-verifier/description.md

```diff
@@ -1,4 +1,5 @@
-Check if a given ISBN-10 is valid.
+The [ISBN-10 verification process](https://en.wikipedia.org/wiki/International_Standard_Book_Number) is used to validate book identification
+numbers.
 
 ## Functionality
```

Comment:
Do any other `description.md`s have a Functionality section?

Reply:
I thought this was odd too. I am not too sure what the original author means - can you think of a better title? :slightly_smiling_face:

Reply:
What about: Move the whole functionality section under the example and rename it to 'Task'?